### ISSUE TITLE
How to Manually Import Plex User into SQlite database?

### ISSUE THREAD

**Issue by kkatayama** [Ombi-app/Ombi#3888]
**Describe the bug**
I recently added a friend to my plex server and Ombi fails to import.

```
Ombi[18134]:       Could not create Plex user since the have no username, PlexUserId: 4547XXXX
```

Is it possible to manually import this user into the Ombi > AspNetUsers table?
If so, how do I generate the `Id`, `SecurityStamp` and `Concurency` stamp fields?

**Sample Entry**
```
                  Id = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
            UserName = xxxx_user
  NormalizedUserName = XXXX_USER
               Email = xxxx@YAHOO.com
     NormalizedEmail = XXXX@YAHOO.COM
      EmailConfirmed = 0
         PhoneNumber =
PhoneNumberConfirmed = 0
    TwoFactorEnabled = 0
          LockoutEnd =
      LockoutEnabled = 1
   AccessFailedCount = 0
               Alias =
            UserType = 2
      ProviderUserId = 4547XXXX
        LastLoggedIn = 2020-11-14 01:08:16.4366428
   MovieRequestLimit = 0
 EpisodeRequestLimit = 0
   MusicRequestLimit = 0
     UserAccessToken =
        PasswordHash =
       SecurityStamp = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
    ConcurrencyStamp = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
            Language =
```

**Logs (Logs directory where Ombi is located)**

Nov 24 22:30:34 mal Ombi[16852]: info: Ombi.Schedule.Jobs.Plex.PlexUserImporter[0]
Nov 24 22:30:34 mal Ombi[16852]:       Could not create Plex user since the have no username, PlexUserId: 4547XXXX

**Desktop (please complete the following information):**
 - OS: Android, Windows, Mac, Linux

**Ombi Version (please complete the following information):**
- Version 4.0.721.0
- Media Server : PLEX
- Database Type: SQLite

**Comment by tidusjar** [comment]
Ombi does not support Plex Home users, there is no way to authenticate them via the Plex.tv API.

**Comment by kkatayama** [comment]
Understood, 

I ended up not having to manually import the user.  I generated a new `INVITE HOME USER` request using the plex web app and after clicked the link I was able to run Ombi's `User Importer` just fine.

Interestingly, all of the users that log into Ombi to make plex requests are **Home Users** and not **Friends**.  
Using Ombi's `custom css` settings I hid the `Sign In with Ombi` button and only display the `Sign in with Plex` button.
All of them have no problem signing into Ombi via plex oauth, which is how I want it and it's awesome :)

*Now I am curious as to how my Ombi setup is able to import and authenticate Plex Home Users when this feature is not currently supported.*

If you would like to see my logs and/or database setting I will gladly share them with you.

> I will note that I do have Ombi running behind an Nginx proxy with certbot managing a Let's Encrypt certificate but I don't think that would affect the import of Plex Home Users.

Thank you for an awesome application!
